web-bundle/src/main/java/com/graphhopper/replica/ReplicaCustomSpeedsBikeTagParser.java

```diff
@@ -1,23 +1,43 @@
 package com.graphhopper.replica;
 
 import com.google.common.collect.ImmutableMap;
+import com.graphhopper.customspeeds.CustomSpeed;
+import com.graphhopper.customspeeds.CustomSpeedId;
 import com.graphhopper.customspeeds.CustomSpeedsUtils;
 import com.graphhopper.customspeeds.DirectedSpeeds;
 import com.graphhopper.reader.ReaderWay;
+import com.graphhopper.routing.ev.EdgeIntAccess;
 import com.graphhopper.routing.ev.EncodedValueLookup;
 import com.graphhopper.routing.util.parsers.BikeAverageSpeedParser;
 import com.graphhopper.util.PMap;
+import org.apache.commons.lang3.NotImplementedException;
+
+import static com.graphhopper.customspeeds.CustomSpeedsUtils.*;
 
 public class ReplicaCustomSpeedsBikeTagParser extends BikeAverageSpeedParser {
-    private final ImmutableMap<Long, DirectedSpeeds> osmWayIdToDirectedMaxSpeeds;
+    private final ImmutableMap<CustomSpeedId<?>, CustomSpeed<?>> idToCustomSpeeds;
 
-    public ReplicaCustomSpeedsBikeTagParser(EncodedValueLookup lookup, PMap configuration, ImmutableMap<Long, DirectedSpeeds> osmWayIdToDirectedMaxSpeeds) {
+    public ReplicaCustomSpeedsBikeTagParser(EncodedValueLookup lookup, PMap configuration,
+                                            ImmutableMap<CustomSpeedId<?>, CustomSpeed<?>> idToCustomSpeeds) {
         super(lookup, configuration);
-        this.osmWayIdToDirectedMaxSpeeds = osmWayIdToDirectedMaxSpeeds;
+        this.idToCustomSpeeds = idToCustomSpeeds;
+    }
+
+    @Override
+    public void handleWayTags(int edgeId, EdgeIntAccess edgeIntAccess, ReaderWay way) {
+        super.handleWayTags(edgeId, edgeIntAccess, way);
```

Comment:
Here's an example of how we throw errors when SEID-based custom speeds have been provided for a non-supported mode, like bike. Note that OSM-based speeds are still handled as they were in `applyMaxSpeed` if they're provided, which is called within `super.handleWayTags()` here

Reply:
do we need to apply these checks to other modes as well?